components/map/layers/tile.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Tile
+page_title: Map Layers - Tile
+description: Discover the Blazor Map Tile Layer and explore the examples.
+slug: components/map/layers/tile
+tags: telerik,blazor,map,layers,tile
+published: True
+position: 3
+---
+
+# Tile Layer
+
+The tile layer works by rendering images to display the actual map that are requested from third-party services that conform to the [Tile Map Service standard](https://en.wikipedia.org/wiki/Tile_Map_Service) and support the [WGS 84 projection standards](https://en.wikipedia.org/wiki/World_Geodetic_System). 
```

Comment:
Please rephrase the first part of this sentence. Its pretty hard to read and understand.